.travis.yml

```diff
@@ -33,26 +33,13 @@ matrix:
       env: TOXENV=py35-alldeps-withcov-posix,codecov-publish
     - python: 3.6
       env: TOXENV=py36-alldeps-withcov-posix,codecov-publish
-    # For now all non-trial tests are in a single job to reduce the time spent
-    # on starting separate jobs.
-    - python: 2.7
-      env: TOXENV=narrativedocs,apidocs,pyflakes,newsfragment,manifest-checker
```

Comment:
I'm not sure it makes sense to move these checks off Travis to CircleCI when the latter imposes a lower usage limit.  If we don't pay CircleCI (or get some free usage from them!) then we won't build any of these linting steps?

Reply:
The limit is only for macOS builds. For Linux, we got 4 concurent linux container which can be used.

So I thought that is a good think to make use of them.

And with this workflow, if the static checker fails, we don't spin the macOS and save some macOS build time.